src/App.tsx

```diff
@@ -21,203 +150,54 @@ export const App: React.FC = () => {
-            <label className="todo__status-label">
-              <input
-                data-cy="TodoStatus"
-                type="checkbox"
-                className="todo__status"
-              />
-            </label>
-
-            <span data-cy="TodoTitle" className="todo__title">Redux</span>
-            <button
-              type="button"
-              className="todo__remove"
-              data-cy="TodoDeleteButton"
-            >
-              ×
-            </button>
-
-            <div data-cy="TodoLoader" className="modal overlay is-active">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
-        </section>
-
-        <footer className="todoapp__footer" data-cy="Footer">
-          <span className="todo-count" data-cy="todosCounter">
-            4 items left
-          </span>
-
-          <nav className="filter" data-cy="Filter">
-            <a
-              data-cy="FilterLinkAll"
-              href="#/"
-              className="filter__link selected"
-            >
-              All
-            </a>
-
-            <a
-              data-cy="FilterLinkActive"
-              href="#/active"
-              className="filter__link"
-            >
-              Active
-            </a>
-            <a
-              data-cy="FilterLinkCompleted"
-              href="#/completed"
-              className="filter__link"
-            >
-              Completed
-            </a>
-          </nav>
+        {todos.length > 0 && (
+          <>
+            <TodoList
+              todos={visibleTodos}
+              onRemove={(todoId: number) => removingTodos(todoId)}
+              deletedTodosIds={deletedTodosIds}
+            />
 
-          <button
-            data-cy="ClearCompletedButton"
-            type="button"
-            className="todoapp__clear-completed"
-          >
-            Clear completed
-          </button>
-        </footer>
-      </div>
+            {isAdding && <TempTodo title={title} />}
 
-      <div
-        data-cy="ErrorNotification"
-        className="notification is-danger is-light has-text-weight-normal"
-      >
-        <button
-          data-cy="HideErrorButton"
-          type="button"
-          className="delete"
-        />
-
-        Unable to add a todo
-        <br />
-        Unable to delete a todo
-        <br />
-        Unable to update a todo
+            <Footer
+              sortBy={sortBy}
+              setSortBy={(sort: Sorting) => setSortBy(sort)}
+              todosCount={visibleTodos.length}
+              onRemove={() => removingTodos()}
+              complitedTodos={completedTodosId}
+            />
+          </>
+        )}
       </div>
+
+      <ErrorNotify
+        isError={isError}
+        onResetError={() => setIsError({
+          status: false,
+          notification: ErrorMessage.None,
+        })}
```

Comment:
move to variable, like in 187 line with `onRemove`. Please fix it ix all over the files

Reply:
corrected